cucumber/api/features/step_definitions/authz_steps.rb

```diff
@@ -82,7 +82,7 @@
 end
 
 Given(/^I create a binary secret value?$/) do
-  @value = Random.new.bytes(16)
+  @value = Random.new.bytes(16).force_encoding(Encoding::UTF_8)
```

Comment:
Check out this RestClient PR: https://github.com/rest-client/rest-client/pull/278

Looks like it used to always use `ASCII-8BIT` for binary data but they changed it to use `UTF-8` instead if the data falls within the appropriate character set. So this is likely an issue w/ RestClient if I'm reading this correctly.